src/HALConfig.cpp

```diff
@@ -46,18 +46,18 @@ int hal_tim_config(TIM_HandleTypeDef *tim, uint32_t t_freq) {
     sConfig.MasterSlaveMode         = TIM_MASTERSLAVEMODE_ENABLE;
 
     if (tim->Instance == TIM1) {
-        __HAL_RCC_TIM1_CLK_ENABLE();
+                __HAL_RCC_TIM1_CLK_ENABLE();
```

Comment:
```suggestion
        __HAL_RCC_TIM1_CLK_ENABLE();
```

Please remove any spaces you added.